internal/common/pagination/pagination.go

```diff
@@ -62,6 +62,22 @@ func CreatePagination(sizeP, pageP *int, total int) (int, *int, LimitOffsetPagin
 	return page, nextPage, NewOffsetFilter(pageSize, offset)
 }
 
+// CreatePagination returns the current page, the page size, and the pagination.LimitOffsetPagination.
+// This method is different approach to the method `CreatePagination` when we don't have the total number of records.
+func CreatePaginationWithoutTotal(sizeP, pageP *int) (int, int, LimitOffsetPagination) {
```

Comment:
We should probably return a struct here and in `CreatePagination` because I'm very confused looking at this function and even the test, what `int, int` represent.

Reply:
Or I like the idea of named returns here.

Reply:
changed to named returns and improve documentations